templates/hound.yml

```diff
@@ -0,0 +1,17 @@
+# See https://houndci.com/configuration for help.
+coffeescript:
+  # config_file: .coffeescript-style.json
+  enabled: false
```

Comment:
I guess what I'm trying to say is that there is no cost to enabling all. The benefit to having everything enabled, is that *if* there is CoffeeScript file, it will be reviewed. Is there ever a need *not* to review CS if it's in the project?

I think CoffeeScript is still the winner according to our guides, so if anything JS should be disabled.